moz_telemetry/sandboxes/heka/analysis/moz_telemetry_new_experiment_monitor.lua

```diff
@@ -42,6 +42,31 @@ alerted = {
 
 new_experiments = {}
 
+-- Parse submission_text as JSON and return the value of "payload.testing".
+-- If it is not present, return false.
+function is_test(submission_text)
+    if not submission_text then
+        return false
+    end
+
+    local submission = cjson.decode(submission_text)
+    if not submission then
+        return false
+    end
+
+    local payload = submission["payload"]
+    if not payload or type(payload) ~= "table" then
+        return false
+    end
+
+    local testing = payload["testing"]
+    if type(testing) ~= "boolean" then
+        return false
```

Comment:
Should we define 'testing' in the schema so it is not an ad-hoc addition and would prevent some derivative like this "payload": {"testing": 1} in the future.  If so 62-67 can be reduced to `return payload.testing`.